app/graphql/resolvers/base_resolver.rb

```diff
@@ -42,6 +42,10 @@ def matching_user(submission, session_id)
       submitted_by_current_session?(submission, session_id)
   end
 
+  def matching_email(submission, user_email)
```

Comment:
Usually, methods that return `true` or `false` have a `?` at the end, so I suggest doing it here.
`def matching_email?`